models/src/main/scala/coop/rchain/models/HashM.scala

```diff
@@ -155,6 +155,7 @@ object HashM extends HashMDerivation {
   implicit val PrivateNamePreviewQueryHash = gen[PrivateNamePreviewQuery]
 }
 
+@SuppressWarnings(Array("org.wartremover.warts.Var"))
```

Comment:
Please only suppress that in `productHash`

Reply:
👍